package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@unocha/hpc-api-core",
+  "version": "0.0.1",
+  "description": "Core libraries supporting HPC.Tools API Backend",
+  "license": "Apache-2.0",
+  "private": false,
+  "scripts": {
+    "check-types": "tsc --noEmit",
+    "prepare": "husky install",
+    "lint-prettier": "prettier -c .",
+    "lint-eslint": "eslint --quiet .",
+    "lint": "yarn lint-prettier && yarn lint-eslint"
+  },
+  "dependencies": {
+    "fp-ts": "^2.6.6",
+    "io-ts": "2.2.9",
+    "knex": "0.21.1",
```

Comment:
Some of these packages can be updated to their latest version, but that could be done in separate PR.

Knex went from `0.21.17` to `0.95.0` and I didn't bother to find out why, but it always seems drastically outdated to me for that reason :)

Reply:
We'll want to check if `hpc_service` still works with this version too then before updating here. I have a feeling that we may need to update things in a few places if so.